rnaseq.snakefile

```diff
@@ -2,7 +2,7 @@ import os
 import pandas as pd
 from lcdblib.snakemake import helpers, aligners
 from lcdblib.utils import utils
-import common
+from lib import common
 
 shell.prefix('set -euo pipefail; ')
```

Comment:
I have been adding TMPDIR stuff here, I don't know if we want to added some code and just have it commented out:

```python
#TMPDIR = os.path.join('/lscratch', os.getenv('SLURM_JOBID'))
#shell.prefix("set -euo pipefail; export TMPDIR={};".format(TMPDIR))
```